### ISSUE TITLE
Hook `filter:user.profileLinks` is deprecated, please use `filter:user.profileMenu` instead.

### ISSUE THREAD

**Issue by Xseba360** [julianlam/nodebb-plugin-canned-responses#10]
```
3/12 14:49 [60] - warn: [plugins/nodebb-plugin-canned-responses] Hook `filter:user.profileLinks` is deprecated, please use `filter:user.profileMenu` instead.
```
NodeBB v1.4.0

**Comment by julianlam** [comment]
Fixed in v2.1.0

**Comment by julianlam** [comment]
7eb45d6c7c57ed1c7274fa1974589d74db1dd96e
